### PR TITLE
remove need for bisect and hipe compliation for now

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
- Prep for use in HLL implementation -> https://github.com/basho/riak_kv/pull/1380

### CODE REVIEW

File: rebar-test.config [hunk 1]
```diff
@@ -2,8 +2,6 @@
 {deps, [
         {basho_stats, "",
          {git, "https://github.com/knutin/basho_stats.git", {branch, "master"}}},
```
Comment (russelldb) [inline rebar-test.config]: lol that we have forked a repo that depends on a fork of one of our repos.

**Comment by russelldb** [comment]
+1 on this. It's a shame that we have to maintain a fork for this. It would be nice if there was some macro or build time option we could use instead.

**Comment by zeeshanlakhani** [comment]
@russelldb, Could upstream something later, but from what i've been told is that we want to apply our forks for such things, and be in control of versioning.

**Comment by zeeshanlakhani** [comment]
@JeetKunDoug @fadushin  for forks like this, how best should we branch these for releases? Thanks.

**Comment by russelldb** [comment]
@zeeshanlakhani fair enough then. Sorry for slow response, for some reason github has stopped sending me any notifications.

**Pull request closed without merging**